core/modules/views/plugins/views_plugin_display.inc

```diff
@@ -1962,11 +1962,11 @@ class views_plugin_display extends views_plugin {
         $form_state['ok_button'] = TRUE;
         break;
       case 'analyze-theme-display':
-        $form['#title'] .= t('Theming information (display)');
-        $output = '<p>' . t('Back to !info.', array('!info' => $this->option_link(t('theming information'), 'analyze-theme'))) . '</p>';
+        $form['#title'] .= t('Theme developer information (display)');
+        $output = '<p>' . t('Back to !info.', array('!info' => $this->option_link(t('Theme developer information'), 'analyze-theme'))) . '</p>';
```

Comment:
Here, and also the other instances of this in this specific file: so we need to capitalise the `T`?

Reply:
Only if it's the first word in a sentence or heading.

Reply:
fixed.